guide/src/documentation/configuration/configure-hermes.md

```diff
@@ -162,9 +162,12 @@ event_source = { mode = 'push', url = 'wss://hello:world@mydomain.com:26657/webs
 
 As of version 1.6.0, Hermes supports the relaying of wasm messages natively. This is facilitated by configuring
 Hermes to use pull-based relaying by polling for IBC events via the `/block_results` RPC endpoint. Set
-the `event_source` parameter to pull mode in `config.toml`:
+the `event_source` parameter to pull mode in `config.toml`, which can be done with either an inline table
+with the `mode` set to `'pull'`, along with the desired polling interval, or instead short-handed by just 
+specifying `'poll'`. 
 
 ```toml
+# The `'poll'` option is an alias for `{ mode = 'pull', interval = '1s' }`
```

Comment:
That is not correct, specifying just `'poll'` raises the following error:

> The Hermes configuration file at path '/Users/romac/.hermes/config.toml' is invalid, reason: parse error: invalid type: string "poll", expected internally tagged enum EventSourceMode for key `chains.event_source` at line 32 column 16


Not sure why we were mentioning that shorthand. Probably a mistake on my part.

Let's just remove mention of `poll` and the (not-working) shorthand.

Reply:
The `poll` is only an alias for `pull`. So the following configurations are equivalent:

* `{ mode = 'pull', interval = '1s' }`
* `{ mode = 'poll', interval = '1s' }`

See: https://github.com/informalsystems/hermes/blob/master/crates/relayer/src/config.rs#L549
As it is mentioned in the guide here https://hermes.informal.systems/documentation/configuration/configure-hermes.html#configuring-support-for-wasm-relaying I think we should either remove the alias and mention from the guide, either specify in the `config.toml` that both the mode `pull` and `poll` are the same.
What do you think?

Reply:
Agreed. Let's only mention `pull` but we can keep the alias internally in case people mistype it.